upgrade.sh

```diff
@@ -7,6 +7,15 @@
 # Python 3.8 or later.
 
 cd "$(dirname "$0")"
+
+NETBOX_VERSION="$(grep ^VERSION netbox/netbox/settings.py | cut -d\' -f2)"
+echo "You are installing (or upgrading to) NetBox version ${NETBOX_VERSION}"
+# Pause if running interactively
+if [ -t 0 ]; then
```

Comment:
In what scenarios might this check return a false positive? I'd rather not risk breaking automated workflows merely to report on the version. Maybe this should be an optional behavior instead?

Reply:
I now realized that checking also `stdout` is needed, otherwise `./upgrade.sh > logfile.txt` will just hang there waiting for input, so that would be `[ -t 0 -a -t 1 ]`.

In that case:
- When run interactively (either directly or via another script), pause is triggered if both `stdin` and `stdout` are on the terminal
- When run via cron job or similar, pause is skipped
- I'm not able to figure out any situations where `stdin` and `stdout` are on the terminal but there is nobody to read and respond (but frankly this is the first time I'm figuring this out)

But yes, if you feel the pause is not desirable after all, let me know, I'll then remove that code. I don't know if adding `--confirm` or similar option would help those that accidentally run `upgrade.sh` with the wrong codebase.

Reply:
Let's not turn this into something more complicated than it needs to be. Let's do this:

* Remove the confirmation prompt and just display the version to which we're upgrading (per the original FR)
* Rebase to the `develop` branch as this is not a v3.7 milestone FR